sklearn/datasets/_california_housing.py

```diff
@@ -50,6 +51,14 @@
 logger = logging.getLogger(__name__)
 
 
+@validate_params(
```

Comment:
I think that this changes should be done in another PR. Could you revert this file.